frappe/www/login.html

```diff
@@ -101,6 +101,15 @@ <h4>{{ _("Javascript is disabled on your browser") }}</h4>
 							</div>
 							{% endfor %}
 						</div>
+						<div class="passwordless_login">
+							{% if passwordless_login %}
+							<div class="login-button-wrapper">
+								<a href="#passwordlessLogin"
+									class="btn btn-block btn-default btn-sm btn-login-option btn-passwordless_login">
+									{{ _("Passwordless Login") }}</a>
```

Comment:
Then I will have to keep some checks to go back to the default login also. Will have to think about it.
Send Magic Link -> Not sure (since it is part of the framework it should be simple)